Sources/TestSupportExecutable/main.swift

```diff
@@ -0,0 +1,56 @@
+import Basic
+import POSIX
+import libc
```

Comment:
Can we have a usage or documentation comment or something in this file which explains all the operations it supports? I imagine it will grow over time, and it would be nice to be able to easily survey what it can do...